component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/x509Certificate/X509CertificateUtil.java

```diff
@@ -335,6 +336,20 @@ private static boolean isUserExists(String userName, AuthenticationContext authe
                 authenticationContext.setProperty(X509CertificateConstants.X509_CERTIFICATE_ERROR_CODE,
                         X509CertificateConstants.USERNAME_CONFLICT);
                 throw new AuthenticationFailedException("Conflicting users with user name: " + userName);
+            } else if (getX509Parameters().containsKey(X509CertificateConstants.LOGIN_CLAIM_URIS)) {
+                for (String multiAttributeClaimUri : getX509Parameters()
+                        .get(X509CertificateConstants.LOGIN_CLAIM_URIS).split(",")) {
+                    String[] usersWithClaim = ((AbstractUserStoreManager) X509CertificateUtil.getUserRealm(userName)
```

Comment:
Assign ((AbstractUserStoreManager) X509CertificateUtil.getUserRealm(userName)
                            .getUserStoreManager()) to a variable and re-use